charon/pkgs/npm.py

```diff
@@ -90,6 +96,9 @@ def handle_npm_uploading(
     valid_dirs = __get_path_tree(valid_paths, target_dir)
 
     prefix_ = remove_prefix(prefix, "/")
+
+    manifest_name, manifest_full_path = write_manifest(valid_paths, target_dir, product)
```

Comment:
I think this step can be skipped with a log if manifest_bucket_name is not specified.